src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -2332,5 +2379,28 @@ private void AddMemberLookupSymbolsInfoInTypeParameter(LookupSymbolsInfo result,
                 AddMemberLookupSymbolsInfoWithoutInheritance(result, baseInterface, options, originalBinder, accessThroughType: type);
             }
         }
+
+        internal void AddImplicitExtensionMemberLookupSymbolsInfoForType(LookupSymbolsInfo result, TypeSymbol type, LookupOptions options, Binder originalBinder)
+        {
+            if (type.ExtendedTypeNoUseSiteDiagnostics is { } extendedType)
+            {
+                type = extendedType;
+            }
+
+            var compatibleExtensions = ArrayBuilder<NamedTypeSymbol>.GetInstance();
+            foreach (var scope in new ExtensionScopes(this))
+            {
+                GetCompatibleExtensions(scope.Binder, type, compatibleExtensions, originalBinder, basesBeingResolved: null);
+
+                foreach (NamedTypeSymbol extension in compatibleExtensions)
+                {
+                    AddMemberLookupSymbolsInfoWithoutInheritance(result, extension, options, scope.Binder, accessThroughType: null);
```

Comment:
>null [](http://example.com/codeflow?start=126&length=4)

Even if the difference is unobservable right now, I think we were passing the right value, given the meaning of the parameter,

Reply:
Restored. Thanks